traffic/data/adsb/opensky_impala.py

```diff
@@ -825,6 +825,10 @@ def history(
             )
 
         elif arrival_airport is not None:
+            if airport is not None:
```

Comment:
I understand that `arrival_airport`/`departure_airport` and `airport` cannot coexist, so I added the relevant error